### PR TITLE
Team Management UI

### PR DESCRIPTION
**Pull request by oguzkocer**
This PR adds most of the Team Management UI for #3286. There are a few stub methods and I am using test data to test the UI. Since this feature is going to be large, we want to merge it to a temporary branch once in a while so the review is easier. I'll also need to switch to work on Me page since it's more urgent and we don't want this branch to go stale without any reviews done on it.

In the PR, I basically added a people page which will eventually show the users in that particular site. I've also added the person detail page which will be accessible once you select a user from the list. You can find screenshots of the current screen below.

device-2015-11-02-150145
device-2015-11-02-150204

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/WordPressDB.java [hunk 1]
```diff
@@ -1940,4 +1942,11 @@ protected void copyDatabase() {
     public boolean hasAnyJetpackBlogs() {
         return SqlUtils.boolForQuery(db, "SELECT 1 FROM " + BLOGS_TABLE + " WHERE api_blogid != 0 LIMIT 1", null);
     }
+
+    /*
+     * This is a stub method. Once implemented it'll retrive a person object for the provided local id
+     */
+    public Person getPersonForLocalTablePostId(long localTablePersonId) {
+        return new Person(4, "oguzkocer", "Oguz", "Kocer", "Oguz", "http://lorempixum.com/76/76", Role.EDITOR);
+    }
```
Comment (maxme) [inline WordPress/src/main/java/org/wordpress/android/WordPressDB.java]: I know it's only a stub method but we should not add anything to `WordPressDB`. Please create a "PersonTable" class and put all db related method there.

**Comment by maxme** [comment]
Looks good, I've some design comments but obviously too early to post them.

**Comment by oguzkocer** [comment]
@maxme I've added the `PersonTable`, thanks for the feedback! For the design comments, I have not yet implemented the full design since we are taking a break from it, I thought we might need to alter it anyway, so this is just something close to it for testing. Should be ready for round 2!

**Comment by maxme** [comment]
:shipit: to `feature/people-management-sync`

**Pull request merged**